addons/turbowarp-player/userscript.js

```diff
@@ -34,11 +34,23 @@ export default async function ({ addon, console }) {
     }
   };
 
+  let showAlert = true;
   while (true) {
-    await addon.tab.waitForElement(".preview .project-buttons", {
+    const seeInside = await addon.tab.waitForElement(".see-inside-button", {
       markAsSeen: true,
       reduxCondition: (state) => state.scratchGui.mode.isPlayerOnly,
     });
+
+    seeInside.addEventListener("click", function seeInsideClick(event) {
+      if (!playerToggled || !showAlert) return;
+
+      if (confirm("Are you sure you want to see inside? Doing this will destroy the TurboWarp player.")) {
+        showAlert = false;
```

Comment:
What does `showAlert` do here? Is this to intentionally not ask the user a second time if they want to destroy the player?

Reply:
@TheColaber Please explain this

Reply:
Correct

Reply:
Okay then, keeping this logic